examples/example_patchextraction.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>documentation is spelled incorrectly. </p><p> </p><p>Please use this link as dataloader will be replaced with wsicore. https://tia-toolbox.readthedocs.io/en/latest/usage.html?highlight=WSIReader.read_rect#tiatoolbox.wsicore.wsireader.WSIReader.read_rect</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Resolved in d04375658bb25e9380f61a35a288fca66a909ce3 commit.